src/site/asn4.rst

```diff
@@ -9,6 +9,24 @@ Assignment #4: 1100010110111010001110110100111110001001
 Task
 ====
 
+Your goal is to compress data such that they take up less space, but can be uncompressed without losing any information.
+You implement Huffman Coding as the compression mechanism. The data will be compressed with an encoding process, and
+uncompressed with a decoding process. Additionally, Huffman Coding can be used as an encryption mechanism with
+pre-shared keys.
```

Comment:
I probably wouldn't mention the encryption use case. Huffman codes produce encryptions that you can break in a fairly straightforward statistical analysis of the compression. Mainly though, it's not part of the assignment, I wouldn't get them too scared.